lib/kubernetes-deploy/options_helper.rb

```diff
@@ -2,11 +2,22 @@
 
 module KubernetesDeploy
   module OptionsHelper
-    def self.default_and_check_template_dir(template_dir)
+    def self.default_and_check_template_dir(template_dir, use_stdin: false)
+      if use_stdin
+        if $stdin.tty?
```

Comment:
Can you explain this check to me? Intuitively I'd expect `$stdin` to be a tty under normal usage, when this command is being invoked from the command line.

Reply:
`$stdin` will be a tty when _not_ reading from a pipe. So in our case, if we specify to use `stdin` but there's no pipe to read from, we'll return an error

Reply:
Changed this to check for length of `ARGV` instead. This gives us more flexibility for extra filename arguments as well as STDIN support via `-`